security/advancedtls/advancedtls_test.go

```diff
@@ -623,3 +623,59 @@ func TestWrapSyscallConn(t *testing.T) {
 			wrapConn)
 	}
 }
+
+func TestOptionsConfig(t *testing.T) {
+	serverPeerCert, err := tls.LoadX509KeyPair(testdata.Path("server_cert_1.pem"),
+		testdata.Path("server_key_1.pem"))
+	if err != nil {
+		t.Fatalf("Server is unable to parse peer certificates, error: %v", err)
```

Comment:
Don't use capital characters in the first letter, because this message could be chained. Same for all the other places.

Reply:
This is a `t.Fatalf`, not a `fmt.Errorf`, so capitals are fine here.

Reply:
Is it better to use `t.Fatalf` or `fmt.Errorf` here?

Reply:
They have very different purposes.  `fmt.Errorf` creates an `error` to be returned to the caller of a function.  `t.Fatalf` causes this test to fail (immediately).

Reply:
Got it, thanks for the explanation!

Reply:
SG, thanks for the clarification Doug! 
Cindy, I just checked for every `t.Fatalf` we did use capitals. To keep them consistent, can you change them back please? Thanks!

Reply:
Sure!